env/src/lib.rs

```diff
@@ -283,6 +280,21 @@ impl LogBuilder {
 }
 
 impl Logger {
+    pub fn new() -> Logger {
+        let mut builder = LogBuilder::new();
+
+        if let Ok(s) = env::var("RUST_LOG") {
+            builder.parse(&s);
+        }
+
+        builder.build()
+    }
+
+    pub fn filter(&self) -> LogLevelFilter {
```

Comment:
`filter` seems ok to me as the returned type is `LogLevelFilter`, but I've always been thrown a bit at least when working out the interaction between `LogLevel`, `LogLevelFilter`, and the various functions that consume them.